### PR TITLE
[copy update] WD-3499 `/managed/apps/kafka` contact link update

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Updated Contact Us links on the page

## QA

- Check out the [demo here](https://ubuntu-com-12857.demos.haus/managed/apps/kafka) and compare it with the changes requested in the [copy doc](https://docs.google.com/document/d/15c89-PyMc-QnSHglG1hf_Tub5lMkEcVYALL0rm5-LvA/edit#).

## Issue / Card

- [WD-3499](https://warthogs.atlassian.net/browse/WD-3499)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-3499]: https://warthogs.atlassian.net/browse/WD-3499?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by petesfrench — COMMENTED** [review]
I have a couple thoughts on this. I think it would be better to just include the same contact form from `/data/kafka` and use the same form ID on this page, rather than redirecting to another page.
I also wonder if the reason why no one is filling out this page's form is because the buttons are missing the class 'js-invoke-modal', which allows the modal to function properly.

**Comment by petesfrench** [comment]
@pedoch Did you hear something back from Rob on this?

**Comment by pedoch** [comment]
@petesfrench Yes, I had a conversation with him and already worked on the changes based on our discussion. I haven't pushed yet because I found something which I need his input on. I intend to reach out to him again on Monday so that I can finalise it.

File: templates/managed/apps/kafka.html [hunk 1]
```diff
@@ -261,7 +261,7 @@ <h4 class="p-heading-icon__title" style="padding-top: .5rem;">Accelerate to mark
       </div>
       <div class="row u-sv3">
         <div class="col-8">
-          <p><a href="/managed/contact-us" class="p-button--positive">Contact us</a></p>
+          <p><a href="#get-in-touch" class="p-button--positive js-invoke-modal">Contact us</a></p>
```
Comment (petesfrench) [inline templates/managed/apps/kafka.html:264]:
```suggestion
          <p><a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a></p>
```

**Review by petesfrench — CHANGES_REQUESTED** [review]
There is another contact-form at the bottom of the page that has a different form ID so will be routed differently, is this what was requested? In most cases it makes more sense to use the same formid (and usually the same form) throughout the page

**Comment by petesfrench** [comment]
@pedoch Yep, sounds good! And updating the links in the doc would be good also, but really could be done by Rob.
I still think we should update the contact-us to link to the same modal as the others.

**Comment by pedoch** [comment]
@petesfrench what exactly do you mean when you say "we should update the contact-us to link to the same modal as the others"? I'm not sure I follow.

**Comment by petesfrench** [comment]
@pedoch the form at the very bottom of the page: 
image

**Comment by pedoch** [comment]
@petesfrench I see what you mean. Yes, that should be included, pushing now.

**Comment by pedoch** [comment]
@petesfrench you can take a look at this again. The links on the copy doc have been updated too.

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**